src/app/globals.css

```
@@ -69,19 +69,40 @@ body {
+  height: 28px;
+  flex: 0 0 28px;
+  display: block;
+}
+.sig-mark-icon-svg {
+  width: 100%;
+  height: 100%;
+  display: block;
+}
+.sig-mark-icon-frame {
+  fill: transparent;
+  stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)